test/test-pages.js

```diff
@@ -118,5 +127,86 @@ describe("#fhirconvert()", function () {
     it("Non func", function () {
       expect(fhir.fhirconvert("NOTAFUNCTION(a+b)", vars)).to.equal(null);
     });
+    it("Non op", function () {
+      expect(fhir.fhirconvert("a$b", vars)).to.equal(null);
+    });
+    it("Op invalid", function () {
+      expect(fhir.fhirconvert("a++b", vars)).to.equal(null);
+    });
+    it("Op invalid 2", function () {
+      expect(fhir.fhirconvert("Patient#HeartRate", vars2)).to.equal(null);
+    });
+    it("Op invalid 3", function () {
+      expect(fhir.fhirconvert("#", vars2)).to.equal(null);
+    });
+    it("Op invalid 4", function () {
+      expect(fhir.fhirconvert("+583", vars2)).to.equal(null);
+    });
+    it("Op invalid 5", function () {
+      expect(fhir.fhirconvert("a+", vars)).to.equal(null);
+    });
+    it("Parenthesis", function () {
+      expect(fhir.fhirconvert(")(a+b)", vars)).to.equal(null);
+    });
+    it("Parenthesis 2", function () {
+      expect(fhir.fhirconvert("CEILING(ABS(a+b)", vars)).to.equal(null);
+    });
+    it("Full Exp", function () {
+      expect(fhir.fhirconvert("CEILING(a+b) ** ABS(c^)", vars)).to.equal(null);
+    });
+    it("Full Exp 2", function () {
+      expect(fhir.fhirconvert("CEILING(a+b) ** ABS(c^2)", vars)).to.equal(("(%a+%b).ceiling().power((%c.power(2)).abs())"));
+    });
+    it("Full Exp 3", function () {
+      expect(fhir.fhirconvert("25+CEILING(a+b)**ABS(c^2)", vars)).to.equal(("25+(%a+%b).ceiling().power((%c.power(2)).abs())"));
+    });
+    it("fun as op test", function () {
+      expect(fhir.fhirconvert("a NOT b", vars)).to.equal((null));
+    });
+  });
+
+  context("Operators", function () {
+    it("and", function () {
+      expect(fhir.fhirconvert("a and b", vars)).to.equal("%a and %b");
+    });
+    it("and 2", function () {
+      expect(fhir.fhirconvert("(a < 3) and b", vars)).to.equal("(%a < 3) and %b");
+    });
+    it("or", function () {
+      expect(fhir.fhirconvert("a or b", vars)).to.equal("%a or %b");
+    });
+    it("equals", function () {
```

Comment:
"it" + description is supposed to make a complete sentence.  it("should ....")....   Nouns go in "describe".  But, it is not the most important thing as long as the test and its output are readable.  If you could follow that going forward, it would be better.  See the example under https://mochajs.org/#interfaces.

Reply:
Okay.